scripts/reader/dc2_static_coadd_reader.py

```diff
@@ -10,36 +10,45 @@
 
 class DC2StaticCoaddCatalog(BaseGenericCatalog):
 
-    _native_filter_quantities = {'tract', 'patch'}
+    _native_filter_quantities = {'tract', 'patch_ra', 'patch_dec'}
```

Comment:
I don't like patch_ra and patch_dec, although I'm fine with dividing them up.  There's nothing that says that patches have to be oriented along RA or Dec.

Reply:
Do you need an int lookup for something?  Is there a reason you can't just have
```
_native_filter_quantities = {'tract', 'patch'}
```
but have patch be a str?

Reply:
@wmwv it surely can. I've updated per your suggestions. The only downside is that people cannot do `(lambda p: p < 2, 'patch_ra')` as a native filter, but I agree with you that this is probably not a common use case.